src/app/user/controller.js

```diff
@@ -16,12 +16,10 @@ const createTokenPayload = user => {
 };
 
 const createUser = async (req, res) => {
-  const { email, username, password, firstName, lastName, role } = req.body;
+  const { email, username, password, role } = req.body;
```

Comment:
Please remove `role` from the request object. 
@Chiagozielam

Reply:
Okay, I'll do this ASAP

Reply:
Please add role back here. I forgot I'd put a condition for the test environment.

Your PR should be good to go after this.